css/tabBar.css

```diff
@@ -102,12 +101,12 @@ body.linux #menu-button {
 
 .tab-item .tab-icon-area {
 	float: right;
+	margin-right: -0.7em;
 }
 .tab-item .tab-icon {
-	font-size: 0.8em;
-	margin-top: -0.1em;
-	padding: 0.66em;
-	vertical-align: middle;
+	margin-top: -2px;
+	font-size: 1em;
+	padding: 0.66em 0.5em;
```

Comment:
It seems like the sizes a bit off; here's a comparison:

<img width="798" alt="Screen Shot 2020-07-16 at 11 18 17 AM" src="https://user-images.githubusercontent.com/10314059/87696225-2465fd00-c756-11ea-8bb3-cf993fb5357b.png">

It feels like the private tab and insecure icons should be a bit smaller than they are, and the close button should be a bit larger.

Reply:
It's because that `add` and `close` are designed smaller than others. Guess I might need to enlarge them in order to make them look the visually same size.
![image](https://user-images.githubusercontent.com/11247099/87700653-3f258980-c7c9-11ea-9331-c282d2aa0cb7.png)